tests/src/tests/mod.rs

```diff
@@ -402,7 +404,14 @@ async fn test_storage_deposit_refund() {
     let balance_after_delete = sandbox.balance(factory.id()).await;
     println!("balance_after_delete: {balance_after_delete}");
     assert_eq!(
-        balance_before_create - balance_after_delete,
-        3_722_264_436_364_300_000_000 // 0.0037222644363643 Ⓝ
+        to_micro_near(balance_before_create - balance_after_delete),
+        0.003_722 // Ⓝ
     );
 }
+
+fn to_micro_near(amount: u128) -> f64 {
```

Comment:
This does convert to NEAR, not microNEAR because you first have the division by 1e18 and then the division by 1e6.

Reply:
Fair enough 👍🏻